pkg/sriov/sriov.go

```diff
@@ -400,7 +400,7 @@ func (s *sriovManager) ResetVFConfig(conf *sriovtypes.NetConf) error {
 	}
 
 	// Restore the original administrative MAC address
-	if conf.MAC != "" {
+	if conf.OrigVfState.AdminMAC != "" {
```

Comment:
please add a check that `conf.OrigVfState` is not nil before we check the AdminMac we can get a null pointer exception here